_includes/app/BasicSample.java

```diff
@@ -0,0 +1,35 @@
+import java.sql.*;
+
+/*
+You can compile and run this example with a command like:
+  javac BasicSample.java && java -cp .:~/path/to/postgresql-9.4.1208.jar BasicSample
+You can download the postgres JDBC driver jar from https://jdbc.postgresql.org.
+*/
+public class BasicSample {
+    public static void main(String[] args) throws ClassNotFoundException, SQLException {
+        // Load the postgres JDBC driver.
+        Class.forName("org.postgresql.Driver");
+
+        // Connect to the "bank" DB on a local cockroach node.
```

Comment:
eh, then you just have to set it later. cockroach does not require a db exist at connection time, only when you use a table in it.